### PR TITLE
Remove pure kubectl Che deployment instructions

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
replace old instructions for deploying single-user Che using pure kubectl https://github.com/eclipse/che/pull/11894 , with approach of using Helm

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/11135

### CODE REVIEW

File: src/main/pages/setup-kubernetes/kubernetes-single-user.adoc [hunk 1]
```diff
@@ -7,28 +7,96 @@ permalink: kubernetes-single-user.html
...
+minikube start --cpus 2 --memory 4096 --extra-config=apiserver.authorization-mode=RBAC
+----
++
+** For Minikube 0.25.2 and lower, use the following command:
++
+----
+minikube start --cpus 2 --memory 4096 --extra-config=apiserver.Authorization.Mode=RBAC
+----
++
+* To deploy role binding, use the following command:
```
Comment (sleshchenko) [inline src/main/pages/setup-kubernetes/kubernetes-single-user.adoc:27]: >  To deploy role binding

I think it makes sense to describe: Which role binding? Why it is needed?

File: src/main/pages/setup-kubernetes/kubernetes-single-user.adoc [hunk 2]
```diff
@@ -7,28 +7,96 @@ permalink: kubernetes-single-user.html
...
+kubectl apply -f ./tiller-rbac.yaml
+----
++
+.. Install `tiller`.
++
+----
+helm init --service-account tiller
+----
++
+* Ensure that you have an NGINX-based Ingress controller. This is the default Ingress controller on Minikube. To start it, use the `minikube addons enable ingress` command.
```
Comment (sleshchenko) [inline src/main/pages/setup-kubernetes/kubernetes-single-user.adoc]: Actually, Che is able to work on Kubernetes if any Ingress controller is there.

**Review by sleshchenko — APPROVED** [review]
LGTM

File: src/main/pages/setup-kubernetes/kubernetes-single-user.adoc (hunk 2 above)
Comment (mkuznyetsov) [reply src/main/pages/setup-kubernetes/kubernetes-single-user.adoc]: ok

**Pull request merged**